source/index.rst

```diff
@@ -280,6 +280,7 @@ Wazuh is a free and open source platform for threat detection, security monitori
    upgrade-guide/index
    user-manual/index
    development/index
+   packaging/index
```

Comment:
Please change the position of this line. It must be between Migrating from ossec and Releases notes.